doc/api/path.md

```diff
@@ -452,10 +464,14 @@ On Windows:
 // returns ['foo', 'bar', 'baz']
 ```
 
+*Note*: This property is *not* read-only and may be modified.
```

Comment:
Not overly useful but still worth mentioning. In fact, it might be worth mentioning explicitly that the path module itself does not use this property.